packages/frontend/src/app/repository/repository.component.spec.ts

```diff
@@ -151,7 +158,9 @@ describe('RepositoryComponent', () => {
     expect(location.path()).toEqual(expectedRoute);
   }));
 
+  /*
```

Comment:
I think we could get rid of this commented out part?

Reply:
Oh yes, will do!